etc/HIProdOfflineConfiguration.py

```diff
@@ -101,8 +101,8 @@
 # maxRunPreviousConfig = 999999 # Last run before era change 08/09/23
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_14_1_5_patch1",
-    'acqEra': {'Run2024J': "CMSSW_14_1_4_patch1"},
+    'default': "CMSSW_15_1_0_patch3",
+    'acqEra': {'Run2025A': "CMSSW_15_1_0_patch3"},
```

Comment:
Should it be `HIRun2025A` instead for acquisition era?
Do we need this line? I see it was removed in last replay PR.

Reply:
I mean in the following PR,
https://github.com/dmwm/T0/pull/5166/files